src/components/MarkdownEditor.vue

```diff
@@ -63,11 +63,11 @@ export default {
     v-spacer
     v-toolbar-items
       v-btn(text, @click="activeTab = 0", :class="{ active: activeTab === 0 }")
-        v-layout(align-center, justify-content, column)
+        v-row.flex-column.align-center.justify-content(no-gutters)
```

Comment:
`v-row.flex-column` makes no sense to me. 

What's the resulting style?

```
display: flex
flex-direction: column
```

What does the row part do?

Reply:
It also kinda upsets me that all of those helper classes add a bunch of `!important` everywhere.

Reply:
Agreed, `!important` is a code smell when it's in library code. Usually indicates that one is subverting the normal rules of CSS that control style precedence, but in Vuetify's case it may just be their way of dealing with a huge amount of complexity and combinatorics. It's an unenviable job...

Reply:
I think ultimately all of these change comes down to, quote "_The grid has been rebuilt modeled after bootstrap. The existing grid still works and needs some slight modifications._". And since bootstrap doesn't have attributes, so stuff got downgraded into helper classes. 😅
But all the `!important` does look scary to me.